README.md

```diff
@@ -1,38 +1,53 @@
 
+The libraries released in the July 2019 preview:
+* [Azure.ApplicationModel.Configuration](/sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md)
 * [Azure.Messaging.EventHubs](/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
 * [Azure.Identity.KeyVault.Keys](/sdk/keyvault/Azure.Security.KeyVault.Keys/Readme.md)
 * [Azure.Identity.KeyVault.Secrets](/sdk/keyvault/Azure.Security.KeyVault.Secrets/Readme.md)
 * [Azure.Storage.Blobs](/sdk/storage/Azure.Storage.Blobs/README.md)
 * [Azure.Storage.Files](/sdk/storage/Azure.Storage.Files/README.md)
 * [Azure.Storage.Queues](/sdk/storage/Azure.Storage.Queues/README.md)
+* [Azure.Core](/sdk/core/Azure.Core/README.md) (shared library)
 
-### Shared libraries
+>NOTE: If you need to ensure your code is ready for production, use one of the stable libraries.
 
-Azure SDK clients use shared libraries implementing retries, logging, transport protocols, authentication protocols, etc.
+### Client: Stable
+Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the preview packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. Stable library directories typicaly contain 'Microsoft.Azure' in their names, e.g. 'Microsoft.Azure.KeyVault'.
 
-[Azure.Core](/sdk/core/Azure.Core/README.md)
+### Management
+Libraries which enable you to provision specific server resources. They are directly mirroring Azure service's REST endpoints. Management library directories typicaly contain the word 'Management' in their names, e.g. 'Microsoft.Azure.Management.Storage'.
 
-### Other services
-
-[Azure.ApplicationModel.Configuration](/sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md)
+## Need help?
+* For reference documentation visit the [Azure SDK for .NET API Reference](http://aka.ms/net-docs).
+* For tutorials, samples, quick starts, and other documentation, go to [Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/azure/).
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-net/issues/new/choose).
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure-net-sdk) or ask new ones on StackOverflow using `azure-net-sdk` tag.
 
 ## Contributing
-
 For details on contributing to this repository, see the [contributing guide](CONTRIBUTING.md).
+
+| Component | Build Status |
+| --------- | ------------ |
+| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/290?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=290&branchName=master) [![Dependencies](https://img.shields.io/badge/dependencies-analyzed-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/dependencies/dependencies.html) |
```

Comment:
Is there a need to clarify which 'Client Libraries' this row relates to? Or do we gloss over that for the sake of simplicity and just use this to refer to track two client libraries?

Reply:
At some point we should clarify. And in general, I think these READMEs should still be improved over time. But I would like to merge as-is as I need to wrap things up before my vacation :-)